### DISCUSSION TITLE
how to delete from a manual install?

### DISCUSSION THREAD

**Discussion by crimsonduelist** [davatorium/rofi#1864] [Q&A]
I m on arch used meson+ninja way
Dont know how to go about uninstalling?
I assume it s smthing like use ninja to uninstall and or remove the cloned directory -> maybe delete some binaries

**Comment by DaveDavenport** [comment]
`ninja uninstall` in the directory you typed `ninja install`

**Comment by crimsonduelist** [comment]
that s what I tried originally but got 
error loading build.ninja no such file or directory

**Reply by DaveDavenport** [reply]
if that was the build you installed, it should uninstall ( I just tested this).

If you took a clean checkout to 'uninstall', try first configuring the project.

**Comment by crimsonduelist** [comment]
yeah I m probably missremembering
I did a meson configure and I got a warning source dir instead of build dir was specified
can I find where the build directory was through meson or ninja directly?

**Comment by crimsonduelist** [comment]
ok nvm tnx, I went into build/ directory... and did a ninja uninstall successfully
thank you again and sry for the inconvenience!
